tool/docker-protoc/Dockerfile

```diff
@@ -5,10 +5,13 @@ FROM    golang:1.18-alpine as builder
 # install deps
 RUN     apk --no-cache add make git go rsync libc-dev openssh docker npm
 # install compilers
-WORKDIR $GOPATH/src/berty.tech/berty
+WORKDIR $GOPATH/src/berty.tech/weshnet
 COPY    go.mod go.sum ./
 RUN     go install -mod=readonly \
+		  github.com/bufbuild/buf/cmd/buf \
```

Comment:
Bad indentation, either you should :
- setup an editor-config plugin in your editor
- edit the `.editorconfig` file in the root of the repo